node/network/protocol/src/grid_topology.rs

```diff
@@ -138,15 +242,39 @@ impl SessionGridTopology {
 	}
 }
 
+/// An entry tracking a session grid topology and some memoized local neighbors.
```

Comment:
It seems that `hunspell` does not know about the word 'memoize', so we should either add this in the dictionary or change the comment to something like this:

```suggestion
/// An entry tracking a session grid topology and some cached local neighbors.
```